exercicioLogica7/src/main/java/org/example/FibonacciService.java

```diff
@@ -0,0 +1,27 @@
+package org.example;
+
+import java.util.Arrays;
+
+public class FibonacciService {
+    private int numero = 9;
+
+    private int[] sequenciaFibo = new int[numero + 1];
+
+    private int x;
+
+
```

Comment:
A lógica esta certa e da pra entender bem. Mas o exercicio pedia para usar while ou do-while. Seria legal já que você fez com classe e método, passar o numero como parametro do seu método calculcaSequenciaFibo:  public int[] calculaSequenciaFibo(int numero).

Reply:
Fiz a alteração de passar número no parâmetro do método. Conferi e esse exercício ainda é com o uso do for, a partir do 8 que é while/do-while. O exercício 9 é continuação desse e possui a inclusão do while/do-while.